### DISCUSSION TITLE
Can I run AUCell with my own list of regulons?

### DISCUSSION THREAD

**Discussion by fransilvion** [aertslab/SCENIC#255] [* General SCENIC questions]
Hi, 

If I already have my own list of TFs and their targets from some external resource (RegNetwork for example), can I use it with AUCell and skip GENIE3 and enrichment step? If yes, how can I do it? 

Also, can I use SCENIC output for pseudotime construction to see how activity of TFs is changing on pseudotime axis?

Thanks

**Comment by s-aibar** [comment]
Dear @fransilvion,

Yes, with AUCell you can use any type of gene set (e.g. TFs and potential target genes, pathway... etc, examples in the [AUCell vignette](https://bioconductor.org/packages/release/bioc/vignettes/AUCell/inst/doc/AUCell.html)). 
Once you have the AUC matrix, you can also choose to continue SCENIC's workflow (e.g. binarization, heatmap... etc) or use it as input for dimensionality reduction methods such as t-SNE/UMAP or pseudotime/trajectory inference. Of course, you will just need to adjust the interpretation of the results according to the input gene-sets...
